plugins/performance-lab/includes/admin/load.php

```diff
@@ -305,7 +305,7 @@ function perflab_install_activate_plugin_callback(): void {
 		wp_die( esc_html__( 'Missing required parameter.', 'performance-lab' ) );
 	}
 
-	$plugin_slug = perflab_sanitize_plugin_slug( wp_unslash( $_GET['slug'] ) );
+	$plugin_slug = perflab_sanitize_plugin_slug( wp_unslash( $_GET['slug'] ) ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- perflab_sanitize_plugin_slug() is a sanitizing function.
```

Comment:
As an alternative to this, the function could be added to `customUnslashingSanitizingFunctions`, but then Plugin Check wouldn't know about it.

Reply:
On the other hand, I don't think Plugin Check handles `phpcs:ignore` comments, so it won't have an effect there.

Related: https://github.com/WordPress/plugin-check/issues/901